www/include/configuration/configObject/traps-mibs/formMibs.php

```diff
@@ -125,7 +125,7 @@ function myDecodeMib($arg)
         $msg .= str_replace("\n", "<br />", $stdout);
         $msg .= "<br />Moving traps in database...";
 
-        $command = "@CENTREONTRAPD_BINDIR@/centFillTrapDB -f '" . $values["tmp_name"]
+        $command = "/usr/share/centreon/bin/centFillTrapDB -f '" . $values["tmp_name"]
```

Comment:
Do not replace this label because it will be automatically replaced by the IC

Reply:
Restored